courses/machine_learning/deepdive2/feature_engineering/solutions/1_bqml_basic_feat_eng_bqml.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<p>Doesn't like we use python at all in this lab, so would this be safe to delete? If yes, I believe we can do the same with the cell directly above this one too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Hi Danielle - Can you clarify which Python code?  The pip install is needed for the BQ library --as I think I recall Evan saying that students can use these notebooks when they are done with class.  The other code is needed to create the project, dataset, and storage bucket.  Thoughts?

Reply:
I don't think the cell with import OS is needed because we don't need the region or bucket anymore because we're not creating a bucket, and the project is inferred from the gcloud config cell above. Let me know if I'm reading it wrong.

Reply:
Agreed. import OS removed from the statement from Lab01.